docs/scripting/callbacks/OnActorStreamIn.md

```diff
@@ -36,10 +36,6 @@ public OnActorStreamIn(actorid, forplayerid)
 
 ## Notes
 
-:::tip
-
-This callback can also be called by NPC.
-
-:::
+<T.TipNPCCallbacks />
```

Comment:
No import for T

Reply:
those who got no imports a line above the TipNPCCallbacks are like that cause it was imported before, check top of that file